lib/bolt/inventory/group2.rb

```diff
@@ -150,6 +150,7 @@ def target_data(target_name)
           resolved_data = resolve_data_keys(target, target_name).merge(
             'name' => target['name'],
             'uri' => target['uri'],
+            'alias' => target['alias'],
```

Comment:
Do we need alias or target_alias in https://github.com/puppetlabs/bolt/blob/6741379142ab9485f5c7f8c0f9cb4d50e2201408/lib/bolt/inventory/group2.rb#L221-L239 ?

Reply:
Looks like adding `alias` to the hash is causing a test to fail at `spec/bolt/inventory/group2_spec.rb:50` since the test isn't expecting the hash to have that value.

Reply:
Its much worse than that unfortunately. For example try this plan out:
```
plan alias_broke(){
  Target.new('uri' => 'localhost', 'target_alias' => 'foo')
  notice(get_target('foo'))
  run_command('whoami', 'foo')
}
```
I think I have a fix, updating tests now.